ioctl/cmd/node/nodereward.go

```diff
@@ -32,13 +32,18 @@ var (
 		config.English: "Query rewards",
 		config.Chinese: "查询奖励",
 	}
+	rewardPoolLong = map[config.Language]string{
+		config.English: "ioctl node reward returns unclaimed and available Rewards in fund pool. TotalUnclaimedRewards is the amount of all delegates that have been issued but are not claimed; TotalAvailableRewards is the amount of balance that has not been issued to anyone.\n\nioctl node [ALIAS|DELEGATE_ADDRESS] returns unclaimed rewards of a specific delegate.",
+		config.Chinese: "ioctl node reward 返回奖金池中的未支取奖励和可获取的奖励. TotalUnclaimedRewards是所有代表已被发放但未支取的奖励的总和; TotalAvailableRewards 是奖金池中未被发放的奖励的总和.\n\nioctl node [ALIAS|DELEGATE_ADDRESS] 返回特定代表的已被发放但未支取的奖励.",
```

Comment:
1. Remove all the “Reward” prefix in the specific command as it is redundant, e.g. ioctl node reward totalUnclaimed, ioctl node reward totalAvailable...
2. For unclaimed rewards of a specific delegate, we should use ioctl node reward unclaimed [ALIAS|DELEGATE_ADDRESS].

Reply:
I change to this:
`~ ioctl node reward pool
TotalUnclaimed: 2821758.422464681337669505 IOTX   TotalAvailable: 32152308.517012998930065838 IOTX`
`~ ioctl node reward unclaimed mhs
io1cjl7x97lp8af4au02z59lyw60f7akgdpqlarmy: 0 IOTX`